eras/dijkstra/impl/cddl/lib/Cardano/Ledger/Dijkstra/HuddleSpec.hs

```diff
@@ -856,3 +874,12 @@ instance HuddleRule "cost_models" DijkstraEra where
           , opt $ idx 3 ==> arr [0 <+ a (huddleRule @"int64" p)]
           , 0 <+ asKey ((4 :: Integer) ... (255 :: Integer)) ==> arr [0 <+ a (huddleRule @"int64" p)]
           ]
+
+instance HuddleRule1 "set" DijkstraEra where
+  huddleRule1 _ = huddleRule1 @"set" (Proxy @ConwayEra)
+
+instance HuddleRule1 "nonempty_set" DijkstraEra where
+  huddleRule1 _ = huddleRule1 @"nonempty_set" (Proxy @ConwayEra)
+
+instance HuddleRule1 "nonempty_oset" DijkstraEra where
+  huddleRule1 _ = huddleRule1 @"nonempty_oset" (Proxy @ConwayEra)
```

Comment:
Let's try and defined huddle rules without even importing any of the previous era types.
```suggestion
instance HuddleRule1 "set" DijkstraEra where
  huddleRule1 _ = maybeTaggedSet

instance HuddleRule1 "nonempty_set" DijkstraEra where
  huddleRule1 _ = maybeTaggedNonemptySet

instance HuddleRule1 "nonempty_oset" DijkstraEra where
  huddleRule1 _ = maybeTaggedNonemptyOset
```